trustchain-cli/src/bin/main.rs

```diff
@@ -189,6 +196,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let resolver = verifier.resolver();
     let mut context_loader = ContextLoader::default();
     match matches.subcommand() {
+        Some(("status", _)) => {
+            // TODO: update the trustchain-api crate.
```

Comment:
Remove comment as currently not required beyond use in this CLI
```suggestion
```

Reply:
done